example/lib/samples/login_sample.dart

```diff
@@ -0,0 +1,95 @@
+        'rememberMe': false,
+      });
+
+  @override
+  Widget build(BuildContext context) {
+    return SampleScreen(
+      title: Text('Login sample'),
+      body: ReactiveFormBuilder(
+        form: buildForm,
+        builder: (context, form, child) {
+          return ListView(
+            children: [
+              ReactiveTextField(
+                formControlName: 'email',
+                validationMessages: (control) => {
+                  ValidationMessage.required: 'The email must not be empty',
+                  ValidationMessage.email:
+                      'The email value must be a valid email',
+                  'unique': 'This email is already in use',
+                },
+                textInputAction: TextInputAction.next,
+                decoration: InputDecoration(
+                  labelText: 'Email',
+                  helperText: '',
+                  helperStyle: TextStyle(height: 0.7),
+                  errorStyle: TextStyle(height: 0.7),
+                ),
+              ),
+              SizedBox(height: 16.0),
+              ReactiveTextField(
+                formControlName: 'password',
+                obscureText: true,
+                validationMessages: (control) => {
+                  ValidationMessage.required: 'The password must not be empty',
+                  ValidationMessage.minLength:
+                      'The password must be at least 8 characters',
+                },
+                textInputAction: TextInputAction.done,
+                decoration: InputDecoration(
+                  labelText: 'Password',
+                  helperText: '',
+                  helperStyle: TextStyle(height: 0.7),
+                  errorStyle: TextStyle(height: 0.7),
+                ),
+              ),
+              Row(
+                children: [
+                  ReactiveCheckbox(formControlName: 'rememberMe'),
+                  Text('Remember me')
+                ],
+              ),
+              SizedBox(height: 16.0),
+              ReactiveFormConsumer(
+                builder: (context, form, child) {
+                  return RaisedButton(
+                    child: Text('Sign Up'),
+                    onPressed: () {
+                      if (!form.valid) {
+                        form.markAllAsTouched();
+                        form.setErrors(form.errors);
+                      }
+
+                      if (form.valid) {
+                        print(form.value);
+                      }
+                    },
+                  );
+                },
+              ),
+              RaisedButton(
+                child: Text('Reset all'),
+                onPressed: () => form.resetState({
+                  'email': ControlState(value: 'johnDoe', disabled: true),
+                  'progress': ControlState(value: 50.0),
+                  'rememberMe': ControlState(value: false),
+                }, removeFocus: true),
+              ),
```

Comment:
If this is the first and simplest sample, them you could remove the **disabled:true** and use it in other sample.